kivy/uix/actionbar.py

```diff
@@ -452,17 +452,17 @@ def _toggle_dropdown(self, *largs):
             # auto_dismiss applies to touching outside of the DropDown
             item.bind(on_release=ddn.dismiss)
 
-    def add_widget(self, item):
+    def add_widget(self, widget, index=0, canvas=None):
```

Comment:
Yeah, agree it's better to simply fix the API and indicate in the docs that it changed. This is 2.0 after all anyway, and `item` was not the correct API anyway.

Reply:
So this would be defined as:
`def add_widget(self, widget, *args, **kwargs):` 
with note in doc stating that `item` is renamed to `widget` to match `Widget.add_widget` method.

Reply:
Yeah!